docs/book/user-guide/tutorials/managing-scheduled-pipelines.md

```diff
@@ -0,0 +1,332 @@
+
+```python
+from zenml.client import Client
+
+# Get the client
+client = Client()
+
+# List all schedules
+schedules = client.list_schedules()
+
+# Find our schedule
+our_schedule = next(
+    (s for s in schedules if s.name == "daily-data-processing"),
+    None
+)
+
+if our_schedule:
+    print(f"Schedule '{our_schedule.name}' created successfully!")
+    print(f"Cron expression: {our_schedule.cron_expression}")
+    print(f"Pipeline: {our_schedule.pipeline_name}")
+else:
+    print("Schedule not found!")
+```
+
+Using the CLI to verify:
+
+```bash
+# List all schedules
+zenml pipeline schedule list
+
+# Filter schedules by pipeline name
+zenml pipeline schedule list --pipeline_id my_pipeline_id
+```
+
+## Step 4: Update the Schedule
+
+Sometimes we need to modify an existing schedule. Since ZenML doesn't support direct schedule updates, we'll need to delete the old schedule and create a new one.
+
+```python
+# First, delete the existing schedule
+client.delete_schedule("daily-data-processing")
+
+# Create a new schedule with updated parameters
+new_schedule = Schedule(
+    name="daily-data-processing",
+    cron_expression="0 10 * * *"  # Changed to 10 AM
+)
+
+# Attach the new schedule to our pipeline
+updated_pipeline = daily_data_pipeline.with_options(schedule=new_schedule)
+
+# Run the pipeline to create the new schedule
+updated_pipeline()
+```
+
+Using the CLI to delete a schedule:
+
+```bash
+# Delete a specific schedule
+zenml pipeline schedule delete daily-data-processing
+```
+
+> **Important**: When updating schedules, you should also delete the
+> corresponding schedule in your orchestrator (Vertex AI in this example). You
+> can do this through the Google Cloud Console or using the orchestrator's API
+> (see below for code example).
+> ZenML's delete command may not always completely remove the underlying
+> orchestrator schedule.
+
+## Step 5: Monitor Schedule Execution
+
+Let's check the execution history of our scheduled pipeline:
+
+```python
+# Get recent pipeline runs
+runs = client.list_pipeline_runs(
+    pipeline_name_or_id="daily_data_pipeline",
+    sort_by="created",
+    descending=True,
+    size=5
+)
+
+print("Recent pipeline runs:")
+for run in runs.items:
+    print(f"Run ID: {run.id}")
+    print(f"Created at: {run.creation_time}")
+    print(f"Status: {run.status}")
+    print("---")
+```
+
+### Monitoring with Alerters
+
+For critical pipelines, [add alerting](https://docs.zenml.io/stacks/alerters) to notify you of failures:
+
+```python
+from zenml.hooks import alerter_failure_hook
+from zenml import pipeline, step
+
+# Add failure alerting to critical steps
+@step(on_failure=alerter_failure_hook)
+def critical_step():
+    # Step logic here
+    pass
+
+@pipeline()
+def monitored_pipeline():
+    critical_step()
+    # Other steps
+```
+
+This assumes you've registered an alerter (like Slack or Discord) in your active stack.
+
+## Step 6: Clean Up
```

Comment:
Link this to the update step. Right now in schedules "Update == Delete + Re-create"